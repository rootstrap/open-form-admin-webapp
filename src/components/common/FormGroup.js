import styled from 'styled-components';

export default styled.div`
  display: flex;
  flex-direction: column;
  width: 60%;
  margin: 0.2em 0;
`;
