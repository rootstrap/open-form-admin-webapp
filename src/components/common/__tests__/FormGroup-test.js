import React from 'react';
import TestRenderer from 'react-test-renderer';

import FormGroup from '../FormGroup';

it('renders correctly', () => {
  const testRenderer = TestRenderer.create(<FormGroup>field</FormGroup>);
  expect(testRenderer.toJSON()).toMatchSnapshot();
});
