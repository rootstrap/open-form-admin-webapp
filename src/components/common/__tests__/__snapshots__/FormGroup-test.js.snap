// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin: 0.2em 0;
}

<div
  className="c0"
>
  field
</div>
`;
